### DISCUSSION TITLE
Prisma throws an error “TypeError: cannot read property findmany of undefined”

### DISCUSSION THREAD

**Discussion by ynsss** [prisma/prisma#3059] [Q&A] [answered]
https://stackoverflow.com/questions/62953062/prisma-throws-error-typeerror-cannot-read-property-findmany-of-undefined

**Comment by ryands17** [comment, marked as answer]
Answered in your previous opened discussion

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
